scripts/create-db.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+createuser -h postgres -U postgres bookbrainz
```

Comment:
@mayhem Do you remember user bookbrainz being used anywhere?
It's not the case in the DB dump script or in any script in this repo

Reply:
This is how other projects do it -- each DB has one or more users to access the DB. The DB dumps are not generating this username yet, since the production DB has not been moved to that user. This is one more step of making things more consistent -- the final step will come when the production DB runs under the bookbrainz user at Hetzner.

Reply:
Roger, thanks.